use_cases/privacy_protection_faces.yml

```diff
@@ -0,0 +1,7 @@
+nodes:
+- input.live
+- model.mtcnn
+- dabble.fps
+- draw.mosaic_bbox
+- draw.legend
+- output.screen
```

Comment:
Will also need to create equivalent ones for the use case tests in our CICD - note that the input and output nodes are different:

https://github.com/aimakerspace/PeekingDuck/tree/dev/tests/use_cases